objective-c-day-4.txt

```diff
@@ -0,0 +1,231 @@
+1)
+#import <Foundation/Foundation.h>
+
+@interface SampleClass : NSObject
+- (NSInteger)findSum:(NSInteger)num1 addNum2:(NSInteger)num2;
+- (NSInteger)evenSum:(NSArray *)input;
+- (void)printHello;
+- (NSMutableArray *)interLeave:(NSArray *)array1 interleave:(NSArray *)array2;
```

Comment:
Two parameters should have clearly different names according objective C nomenclature.